environments/gcp/flyte-core/ingress.tf

```diff
@@ -0,0 +1,82 @@
+resource kubernetes_namespace "flyte-ns" {
+  metadata {
+  name = "flyte"
+  }
+depends_on = [ module.gke ]
+}
```

Comment:
```
resource helm_release "flyte-core" {
  namespace = "flyte"
  create_namespace = true
```

Is this required given that we create the namespace with the `helm_release`?

Reply:
We need the `flyte-secret-tls` Secret to be present in the ns before the release is installed. In theory, the cert-manager controller should still reconcile even if, after a first run, it doesn't find the secret. If we set the Helm release to depend on the secret it could create a chicken-egg problem with the namespace creation.